src/tests/ftest/server/metadata.py

```diff
@@ -16,21 +15,22 @@
 from thread_manager import ThreadManager
 
 
-def run_ior_loop(manager, uuids):
-    """IOR run for each UUID provided.
+def run_ior_loop(test, manager, num_run):
+    """Run IOR multiple times.
 
     Args:
         manager (str): mpi job manager command
-        uuids (list): list of container UUIDs
+        num_run (int): number of times to run IOR
```

Comment:
What do you think about something like `container_qty`, `repetitions`, or `loops`?

Reply:
I changed to `loops`